webpack.config.js

```diff
@@ -112,7 +112,7 @@ const webViewConfig = {
             loader: "file-loader",
             options: {
               name: "[name].[ext]",
-              outputPath: "fonts/",
+              outputPath: "fonts",
             },
```

Comment:
If we remove this dir option then wouldn't this solve this issue `src: url("./codicon.ttf?669d352dfabff8f6eaa466c8ae820e43") format("truetype");` ?

Since it will then output to `./` dir for the webview i.e., `dist/views`

Reply:
Still same issue of encoding using `vscode.asWebviewUri()` and the font location will try to be resolved against original location (~node_modules/vscode-codicons/dist/codicon.ttf since it is where codicon.css is) with `resolve-url-loader` or in the same location as entry file : ${viewDir}/main.ts if resolve-url-loader is ignored.